### PR TITLE
fix: Equalizing dependency versions

### PR DESCRIPTION
**Pull request by evertonfraga**
This PR equalizes package versions. 

- On the developer side, It will enable us to use Lerna's automatic dependency hoisting, that installs common dependencies only once.
- On the CI side, it would make overall cache size smaller.
- On the project quality and maintainability, adds consistency.

Remaining work that is being done (or will be done) in other PRs:
- update MPT to 4.0 (#787)
- remove `async` dependency #779 

Related: #730, #770.

### CODE REVIEW

**Comment by jochem-brouwer** [comment]
LGTM. Did you use any automated software to equalize these dependencies? 

Is it an idea to centralize (i.e. in the root) the packages which multiple packages in the monorepo use?

**Comment by evertonfraga** [comment]
@jochem-brouwer thanks for the review. 

Although moving dependencies to the root package.json would work in our machines and CI, that would not work whenever people installed individual libraries. We're implementing an ESLint check that helps us detecting implicit dependencies, and we'll start using them after releasing ethereumjs-config.


> LGTM. Did you use any automated software to equalize these dependencies?

I normalized those manually, doing some regular expressions and find/replaces. 

And I didn't want to update them all to latest major, because that requires more time investment, and my goal for now is just to simplify the dependency tree.

Regarding failing checks: linting problems 😣

**Comment by evertonfraga** [comment]
Right, this is now ready for review again. I was using the same lint from another PR 😄

File: packages/account/package.json [hunk 1]
```diff
@@ -37,26 +37,26 @@
...
-    "rlp": "^2.2.1",
+    "rlp": "^2.2.3",
     "safe-buffer": "^5.1.1"
   },
   "devDependencies": {
-    "@ethereumjs/config-nyc": "^1.1.0",
-    "@ethereumjs/config-prettier": "^1.1.0",
-    "@ethereumjs/config-tsc": "^1.1.0",
-    "@ethereumjs/config-tslint": "^1.1.0",
+    "@ethereumjs/config-nyc": "^1.1.1",
```
Comment (holgerd77) [inline packages/account/package.json:44]: For my understanding: this is just a cosmetic change, right? Or does this make a difference on CI caching or something?

Comment (evertonfraga) [reply packages/account/package.json:44]: @holgerd77 Even though the version ranges result in the same version being used, it is a requirement for `lerna bootsrap --hoist` to work properly. See big chunk of warnings described in #730.

Comment (evertonfraga) [reply packages/account/package.json:44]: or more explicitly: the repo must use the same exact versions so they get properly hoisted.

**Pull request merged**